homeworks/james.bond_JamesBond/0-intro/README.MD

```diff
@@ -0,0 +1,2 @@
+node - 999
+npm - 0001
```

Comment:
Hello, @MZabolotnev please add the path to node as described in the https://github.com/MastersAcademy/fe-course-basic-2023/issues/3#issue-1931677593 point 6